src/dlgTriggerEditor.cpp

```diff
@@ -2406,69 +3274,70 @@ void dlgTriggerEditor::addTimer(bool isFolder)
 void dlgTriggerEditor::addVar(bool isFolder)
 {
     saveVar();
-    QString name;
-    mpVarsMainArea->key_type->setCurrentIndex(0);
+    mpVarsMainArea->comboBox_variable_key_type->setCurrentIndex(0);
     if (isFolder) {
         // Edbee doesn't have a readonly option, so I'm using setEnabled
         mpSourceEditorEdbee->setEnabled(false);
-        mpVarsMainArea->var_type->setDisabled(true);
-        mpVarsMainArea->var_type->setCurrentIndex(4);
-        mpVarsMainArea->lineEdit_var_name->setText("");
-        mpVarsMainArea->lineEdit_var_name->setPlaceholderText("Table name...");
+        mpVarsMainArea->comboBox_variable_value_type->setDisabled(true);
+        mpVarsMainArea->comboBox_variable_value_type->setCurrentIndex(4);
+        mpVarsMainArea->lineEdit_var_name->setText(QString());
+        mpVarsMainArea->lineEdit_var_name->setPlaceholderText(tr("Table name..."));
 
         clearDocument(mpSourceEditorEdbee, QLatin1Literal("NewTable"));
-        name = "";
     } else {
         // Edbee doesn't have a readonly option, so I'm using setEnabled
         mpSourceEditorEdbee->setEnabled(true);
-        mpVarsMainArea->lineEdit_var_name->setText("");
-        mpVarsMainArea->lineEdit_var_name->setPlaceholderText("Variable name...");
-        mpVarsMainArea->var_type->setDisabled(false);
-        mpVarsMainArea->var_type->setCurrentIndex(0);
-        name = "";
+        mpVarsMainArea->lineEdit_var_name->setText(QString());
+        mpVarsMainArea->lineEdit_var_name->setPlaceholderText(tr("Variable name..."));
+        mpVarsMainArea->comboBox_variable_value_type->setDisabled(false);
+        mpVarsMainArea->comboBox_variable_value_type->setCurrentIndex(0);
     }
+
     QStringList nameL;
-    nameL << name;
+    nameL << QString();
```

Comment:
`name` is an empty string no matter which branch is taken in prior `if()` of old code.